File: PHP/Exercicio007.php
<!DOCTYPE html>
<html>

<body>
 <h4>Verificar senha</h4>
 <form method="post" action="Exercicio007.php">
    <legend>Informe a senha</legend>
    <input type="text" name="palavra" id="palavra">
    <button>Verificar</button>
 </form>
<?php
  if(isset($_POST['palavra'])){
    $palavra = $_POST['palavra'];
    $senha = "123456";
    if(strcmp($palavra, $senha) == 0){
        echo "<p style = 'color:#AA0000'><strong>Acesso garantido</strong></p>";
    }
    var_dump($palavra);
  }
?>
</body>
</html>

File: JAVA/Exercicio007.java
package main;

import javax.swing.JOptionPane;

public class Exercicio007 {

	public static void main(String[] args) {
		String senha = "123456", palavra;
		
		palavra = JOptionPane.showInputDialog("Entre com a senha:");
		
		if(palavra.equals(senha)) {
			JOptionPane.showMessageDialog(null, "Acesso Garantido!");
		}

	}

}
